import_handlers.js

```diff
@@ -0,0 +1,154 @@
+const fs = require('fs');
+const moment = require('moment');
+const xml2js = require('xml2js');
+//const logger = log4js.getLogger('<filename>');
+
+class Transaction {
+    constructor(date, from, to, narrative, amount) {
+        this.date = date;
+        this.from = from;
+        this.to = to;
+        this.narrative = narrative;
+        this.amount = parseFloat(amount);    
+    }
+}
+
+class Account {
+    constructor(name) {
+        this.name = name;
+        this.transactionRefs = [];
+    }
+
+    calcBalance() {
+        var running_total = 0;
+        let account_name = this.name;
+        this.transactionRefs.forEach(function(t) {
+            if (t.to === account_name) {
+                running_total += t.amount;
+            } else {
+                running_total -= t.amount;
+            }
+        });
+        return running_total;
+    }
+}
+
+function import_csv_data(inputPath, accounts) {
+    let transactions = [];
+
+    const csv_out = fs.readFileSync(inputPath, 'utf8');
+    const lineSplit = csv_out.split('\n');
+
+    for (var i=1; i<lineSplit.length-1; i++) {
+        // Start at row 1, as row 0 contains the column headings
+        // End one before the final element, since the last array item is a blank line
+        const transaction = lineSplit[i].split(',');
+
+        // Extract the fields
+        const date = transaction[0];

```

Comment:
You can use destructuring here to move these all into one (shorter) line:

    const [date, from, to, narrative, amount] = transaction;